lib/wrappers/openssl.nim

```diff
@@ -93,6 +93,9 @@ else:
 
 import dynlib
 
+{.pragma: lcrypto, cdecl, dynlib: DLLUtilName, importc.}
```

Comment:
(most of) these should be `raises: [], noSideEffect` as well - don't remember if SSL has globals or not which would affect side effects